polylogue/core/configuration.py

```
@@ -49,10 +52,36 @@ class Defaults:
 @dataclass
 class AppConfig:
     defaults: Defaults = field(default_factory=Defaults)
+    drive: Optional["DriveConfig"] = None
```

Ruff (0.15.9)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)